udata/core/dataset/models.py

```diff
@@ -824,8 +828,7 @@ def next_update(self):
         else:
             return self.last_update + delta
 
-    @cached_property
-    def quality(self):
+    def compute_quality(self):
```

Comment:
I think computing `update_fulfilled_in_time` at save time doesn't make sense though

Reply:
Yes… Don't know what to do about this…

Reply:
Fixed in 06fa31458